app/experimenter/nimbus-ui/src/lib/getConfigLabel.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Ah sorry I made this PR slightly longer than it needed to be, I pulled this out into its own file because we're already using it in two places and I thought I also needed it on the metrics page. Didn't end up using it since probe sets from the config don't have a `label`, but would like to leave it since we use this in the `Summary` and `TableHighlightsOverview` components.